.github/workflows/llvm-project-epoch-one.yml

```diff
@@ -27,15 +27,12 @@ jobs:
       - name: Test statically linked clang
         run: bash ci/test-clang.sh
 
-      - name: Login to ghcr.io
+
+      - uses: ./.github/actions/push-container
         if: ${{ github.repository_owner == 'ClangBuiltLinux' && github.event_name != 'pull_request' && github.ref_name == 'main' }}
-        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Push image to ghcr.io
```

Comment:
Can we keep this name? Otherwise, it will just say `./.github/actions/push-container`, which isn't particularly descriptive to me.